### PR TITLE
Add xiaomi miio troubleshooting

### PR DESCRIPTION
**Pull request by starkillerOG**
## Proposed change

Add xiaomi miio troubleshooting, related to https://github.com/home-assistant/core/issues/81475

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by starkillerOG** [comment]
Can someone approve/merge this?

File: source/_integrations/xiaomi_miio.markdown [hunk 1]
```diff
@@ -73,6 +73,14 @@ you used in the Xiaomi Home App (where you initially setup the device). There ar
 [this page](https://www.openhab.org/addons/bindings/miio/#country-servers) for
 the server to use for each country.
 
+## Troubleshooting
+
+[Detailed troubleshooting steps are here](https://python-miio.readthedocs.io/en/latest/troubleshooting.html#troubleshooting)
```
Comment (frenck) [inline source/_integrations/xiaomi_miio.markdown]: We should not point people to libraries in our documentation.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/xiaomi_miio.markdown (hunk 1 above)
Comment (starkillerOG) [reply source/_integrations/xiaomi_miio.markdown]: I removed that statement.
I hope it is alright to have the `more info` links at the end of each problem in the list?

File: source/_integrations/xiaomi_miio.markdown [hunk 2]
```diff
@@ -73,6 +73,13 @@ you used in the Xiaomi Home App (where you initially setup the device). There ar
 [this page](https://www.openhab.org/addons/bindings/miio/#country-servers) for
 the server to use for each country.
 
+## Troubleshooting
+
+The most common problems are:
+- Xioami Miio devices do not communicate accros subnets/VLANs due to the source address of the UDP packet not belonging to the subnet of the device itself, [more information and solutions](https://python-miio.readthedocs.io/en/latest/troubleshooting.html#discover-devices-across-subnets).
+- Roborock vacuums need to be connected to the Xiaomi Home app, not the roborock app, [more information](https://python-miio.readthedocs.io/en/latest/troubleshooting.html#roborock-vacuum-not-detected).
+- Blocking the network access to the device is known to cause intermittend connection issues due to the device internal software hanging and a watchdog restarting the internal software, [more information](https://python-miio.readthedocs.io/en/latest/troubleshooting.html#intermittent-connection-issues-timeouts-xiaomi-vacuum).
```
Comment (frenck) [inline source/_integrations/xiaomi_miio.markdown]:
```suggestion
- Xiaomi Miio devices do not communicate across subnets/VLANs due to the source address of the UDP packet not belonging to the subnet of the device itself, [more information and solutions](https://python-miio.readthedocs.io/en/latest/troubleshooting.html#discover-devices-across-subnets).
- Roborock vacuums need to be connected to the Xiaomi Home app, not the Roborock app, [more information](https://python-miio.readthedocs.io/en/latest/troubleshooting.html#roborock-vacuum-not-detected).
- Blocking the network access to the device is known to cause intermittent connection issues due to the device's internal software hanging and a watchdog restarting the internal software, [more information](https://python-miio.readthedocs.io/en/latest/troubleshooting.html#intermittent-connection-issues-timeouts-xiaomi-vacuum).
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by starkillerOG** [comment]
@frenck thanks for the spelling checks!

**Review by frenck — APPROVED** [review]
Thanks, @starkillerOG 👍

**Pull request merged**